operator/v1/0000_50_ingress-operator_00-ingressoperator.crd.yaml

```
@@ -0,0 +1,20 @@
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    api-approved.openshift.io: https://github.com/openshift/api/pull/1300
+    include.release.openshift.io/ibm-cloud-managed: "true"
+    include.release.openshift.io/self-managed-high-availability: "true"
+    include.release.openshift.io/single-node-developer: "true"
+  name: ingressoperator.operator.openshift.io
```

This should be `<plural>.<group>`
```suggestion
  name: ingressoperators.operator.openshift.io
```